src/components/user/UserProfile.js

```diff
@@ -47,7 +47,7 @@ export default function PersonalProfile() {
       <Link to="/updateUserInfo">
         {' '}
         {/* Specify the path where you want to redirect */}
-        <Button color="warning" style={{ backgroundColor: 'lightgrey' }}>
+        <Button color="success" style={{ backgroundColor: 'lightgrey' }}>
```

Comment:
![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/f05243de-1b76-4eba-a42c-2e61d088fed8)

this color is a bit difficult to read in dark mode. Maybe we just use the default colour set here? Or use a different background colour. Or create some custom colors in the theme if you refer: https://stackoverflow.com/questions/46486565/material-ui-customize-button-color

Reply:
@akosasante, I totally agree, these styles need to be changed, I noted it to create a separate PR, at the moment I will leave the default color